added endTask function; added D-score calculation and display;

yago1994 opened the pull request:
# Issues
- Functions are not firing or printing to the console

yago1994 committed:
added endTask function; added D-score calculation and display;

Not firing;

Wiza-Munthali committed:
D Score Displayed

Converted the iatExtension to task and used the API Manager to control the sequence of what appears. Added a new script to display the score at the end of the tasks.

Wiza-Munthali commented:
@yago1994 the D score is being shown and added the feedback right below it for extra context. 

Question: is the D score supposed to be calculated in a different way from the d score provided by the iat? Because I am currently using the d score provided from the iat extension and not calculating it with your functions.

yago1994 commented:
Let’s use the D-score provided by the IAT because that’s what gets used for the test, so that’s alright.

However let’s remove the feedback so as not to bias participants to perform differently in the next round of the test. The page should just display the score

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Wiza Munthali ***@***.***>
Sent: Wednesday, July 17, 2024 3:32:27 AM
To: AppHatchery/Gender-career-IAT ***@***.***>
Cc: SantiagoA ***@***.***>; Mention ***@***.***>
Subject: Re: [AppHatchery/Gender-career-IAT] added endTask function; added D-score calculation and display; (PR #2)


@yago1994<https://github.com/yago1994> the D score is being shown and added the feedback right below it for extra context.

Question: is the D score supposed to be calculated in a different way from the d score provided by the iat? Because I am currently using the d score provided from the iat extension and not calculating it with your functions.

—
Reply to this email directly, view it on GitHub<https://github.com/AppHatchery/Gender-career-IAT/pull/2#issuecomment-2232626106>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AD5EOKDGCP4RM7BSLYFQ5OLZMYMYXAVCNFSM6AAAAABK4SCFSOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEMZSGYZDMMJQGY>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Wiza-Munthali committed:
Update score.js

Removing the feedback from displaying

Wiza-Munthali commented:
@yago1994 Done
